app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -242,58 +255,29 @@ def index_organization
     end
 
     api :GET, "/systems/:system_id/subscriptions/available", N_("List available subscriptions"), :deprecated => true
-    param :system_id, String, :desc => N_("UUID of the system"), :required => true
-    param :match_system, :bool, :desc => N_("Return subscriptions that match system")
-    param :match_installed, :bool, :desc => N_("Return subscriptions that match installed")
-    param :no_overlap, :bool, :desc => N_("Return subscriptions that don't overlap")
-    def available_system
+    param :system_id, String, :desc => N_("UUID of the content host"), :required => true
+    param :match_system, :bool, :desc => N_("Return subscriptions that match a content host")
+    param :match_installed, :bool, :desc => N_("Return subscriptions that match installed products")
+    param :no_overlap, :bool, :desc => N_("Return subscriptions that are not attached to covered products")
```

Comment:
https://access.redhat.com/documentation/en-US/Red_Hat_Subscription_Management/1/html-single/RHSM/index.html
From the docs:
The All Available Subscriptions tab shows all of the subscriptions that are available to the system. The default displays only subscriptions that are compatible with the hardware, but these can be filtered to show any subscriptions which match the hardware, any subscriptions which match installed products, only subscriptions which do not overlap with a currently-attached subscription, or any subscription which matches a given string.